doc/TRY.md

```diff
@@ -109,6 +109,9 @@ Liana does not support "hot keys" at the moment. It needs a connection to a sign
 signing transactions.
 
 We will be using a [Specter](https://github.com/cryptoadvance/specter-diy) simulator.
+(In case of compatibility issues with your computer, or if you want to
+test with another device or simulator please read the
+[signing devices documentation](./signing_devices.md))
```

Comment:
I don't think the Speculos mention should go into the `signing_devices.md` document (or it would need a larger refactoring of the doc). Let's just have it directly here instead.

> In case the Specter DIY simulator is not usable on your machine (for instance we've got reports it is not on any >M1 Mac), you can use the Ledger simulator "Speculos".

And link to both the Github and the doc at https://speculos.ledger.com/ please.